client/src/components/Tickets/TicketsLayout.scss

```diff
@@ -6,19 +6,30 @@
   font-family: monospace;
 }
 
-.tickets-view {
-  /* Ticket Number */
-  th:nth-child(1),
-  td:nth-child(1) { min-width: 5em; }
-  /* Subject */
-  th:nth-child(2),
-  td:nth-child(2) { width: 80%; }
-  /* Date */
-  th:nth-child(3),
-  td:nth-child(3) { width: 20%; }
-  /* Status */
-  th:nth-child(4),
-  td:nth-child(4) { min-width: 10em; }
+/* WARNING: Custom standard breakpoint: 1679px / 1680px */
```

Comment:
The introduction of `table-layout: fixed` made `min-width` ineffectual, so I used percentages (that add to 100%), and I changed those percentages depending on screen width. Also, I simplified the syntax (because I needed to add so much code). Not every column needs a percentage, but I lost to my OCD.